src/main/java/at/ac/fhcampuswien/se_booking/dto/currency/ConvertCarPriceDTO.java

```diff
@@ -0,0 +1,7 @@
+package at.ac.fhcampuswien.se_booking.dto.currency;
+
+public record ConvertCarPriceDTO(
+        Long carId,
```

Comment:
CarId is not Long anymore, it is a String

Reply:
changed it to string